mxnet/inference/docker/1.9.0/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,123 @@
+FROM ubuntu:20.04
```

Comment:
can you install these additional packages as well as they are needed https://github.com/aws/deep-learning-containers/pull/1152/files#diff-2c0f9a3b834a5df582fe212170d0abc5f7b80bff8ea915ad43d655aabd0a47caR180

Reply:
sure, are they needed just for training? or also for inference? because in the pr I see only for training.